ros_ws/src/autonomous/include/emergency_stop.h

```diff
@@ -12,43 +12,31 @@
 #include <ros/console.h>
 #include <ros/ros.h>
 
+constexpr float DEG_TO_RAD = M_PI / 180.0;
+
 constexpr const char* TOPIC_LASER_SCAN = "/scan";
 constexpr const char* TOPIC_EMERGENCY_STOP = "/emergency_stop";
 
+constexpr float SAMPLE_ANGLE = 6 * DEG_TO_RAD;
+constexpr float RANGE_THRESHOLD = 0.3;
+
+constexpr float MAX_RANGE = 30;
```

Comment:
Is there a reason why the max range was changed from 4 to 30?

Reply:
This is the value we use in `wall_following.h` and is closer to the actual maximum range of the Lidar. I think it is "more correct" to detect a distant wall rather than no wall, even thouh neither would trigger an emergency brake.